stdlib/LinearAlgebra/src/dense.jl

```diff
@@ -489,6 +489,10 @@ julia> exp(A)
 exp(A::StridedMatrix{<:BlasFloat}) = exp!(copy(A))
 exp(A::StridedMatrix{<:Union{Integer,Complex{<:Integer}}}) = exp!(float.(A))
 
+Base.:^(b::Number, A::AbstractMatrix) = exp(log(b)*A)
```

Comment:
This could actually call `exp!(log(b)*A)` since the multiplication returns a new array.